authentication/views.py

```diff
@@ -112,17 +112,11 @@ def get_social_auth_types(request):
         UserSocialAuth
         .objects
         .filter(user=request.user)
-        .values('provider', 'user__email')
+        .values_list('provider', flat=True)
+        .distinct()
     )
-    serialized_social_auths = [
-        {
-            'provider': social_auth['provider'],
-            'email': social_auth['user__email']
-        }
-        for social_auth in social_auths
-    ]
     return Response(
-        data=serialized_social_auths,
+        data=social_auths,
```

Comment:
Suggestion: keep this as a list of objects in case we need to return additional properties in the future (probably a fair chance at that).

Reply:
Yeah, makes sense. I'll just leave it as a list of objects with `provider` as the only key